src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1395,6 +1395,8 @@ public void restartServer(boolean buildContainer) throws PluginExecutionExceptio
                 }
             }
         }
+        // suppress install feature warning
+        System.setProperty("skipBetaInstallFeatureWarning", "true");
```

Comment:
Instead of defining "true" or "false" which can be prone to typos, considering using something like `Boolean.TRUE.toString()`